docker/micro-cluster-lab/script_files/bootstrap.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+hdfs namenode -format
+service ssh start
+if [ "$HOSTNAME" = node-master ]; then
+    start-dfs.sh
+    start-yarn.sh
+    #start-master.sh
+    cd /root/lab
+    /opt/hadoop/bin/hadoop fs -put /spark_volume/spark_example.txt /spark_example.txt
+    /opt/spark/bin/spark-submit --class Example --master yarn /spark_volume/java-spark-1.0-SNAPSHOT.jar /spark_example.txt
```

Comment:
(question) do we really want to execute this sample code each time we start up our cluster?

Reply:
I think we don't, deleted it